src/Tribe/Views/V2/Hooks.php

```diff
@@ -713,4 +715,28 @@ public function print_single_json_ld() {
 
 		$this->container->make( Template\JSON_LD::class )->print_single_json_ld();
 	}
+
+	/**
+	 * Add views stylesheets to customizer styles array to check.
+	 * Remove unused legacy stylesheets.
+	 *
+	 * @param array $sheets Array of sheets to search for.
+	 *
+	 * @return array Modified array of sheets to search for.
+	 */
+	public function customizer_inline_stylesheets( $sheets, $css_template ) {
+		$v2_sheets = [
+			'tribe-events-views-v2-skeleton',
+			'tribe-events-views-v2-full',
+		];
+
+		// Unenqueue legacy sheets.
+		if ( ( $keys = array_keys( $sheets, 'tribe-events-calendar-style' ) ) !== [] ) {
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: 9f8c7db739dcf9dc1e278116c833bdbd</sub>